src/backend/src/routes/projects.routes.ts

```diff
@@ -81,5 +81,25 @@ projectRouter.post(
   validateInputs,
   ProjectsController.createMaterial
 );
+projectRouter.post(
+  '/bom/material/:wbsNum/edit',
```

Comment:
apologies, this should be :materialId, and then you should use that to fetch the material/ check if it exists in the service method. You should be able to get the wbsNum from that material that you fetch, it shouldn't be changing after a material has been created